snippets/logging-loki-statement-snip.adoc

```diff
@@ -0,0 +1,11 @@
+// Text snippet included in the following assemblies:
+//
+//
+// Text snippet included in the following modules:
+//
+//
+:_mod-docs-content-type: SNIPPET
+
+Loki is a horizontally scalable, highly available, multi-tenant log aggregation system offered as a log store for the {logging}. OpenShift {logging-uc} provides Loki as a GA log store that can be visualized with the OpenShift {ObservabilityShortName} UI. The Loki configuration provided by OpenShift {logging-uc} is a short-term log store designed to enable users to perform fast troubleshooting with the collected logs. For that purpose, the OpenShift {logging-uc} configuration of Loki has short-term storage, and is optimized for very recent queries. For long-term storage or queries over a long time period, users should look to log stores external to their OpenShift cluster.
```

Comment:
`Loki is a horizontally scalable, highly available, multi-tenant log aggregation system offered as a log store for {logging}. {logging-uc} {for} provides Loki as a GA log store that can be visualized with the OpenShift {ObservabilityShortName} UI. The Loki configuration provided by OpenShift {logging-uc} is a short-term log store designed to enable users to perform fast troubleshooting with the collected logs. For that purpose, the {logging} {for} configuration of Loki has short-term storage, and is optimized for very recent queries. For long-term storage or queries over a long time period, users should look to log stores external to their cluster.`

Reply:
Thanks for the suggestion @libander. @jeana-redhat made the change suggested by Libby. :)